meson.build

```diff
@@ -15,7 +15,7 @@ project(
     'cpp_std=c++11',
     'warning_level=1'],
   license : 'Public Domain',
-  meson_version : '>= 0.49.0')
+  meson_version : '>= 0.54.0')
```

Comment:
The fs module is available since 0.53.0, am I overlooking something else that needs 0.54.0?

Reply:
I'm not sure -- when i ran the build meson complained that I should be requiring 0.54.0

I'll do another commit (for the join_paths suggestion, thanks!) and try 0.53.0 then

Reply:
In that case it's probably correct -- I haven't checked out the PR and tried it myself -- it should list which feature requires 0.54 though.

Reply:
looks like the 'name' member function was introduced in 0.54.0

```
WARNING: Project specifies a minimum meson_version '>= 0.53.0' but uses features which were added in newer versions:
 * 0.54.0: {'name'}
```